File: .NET/library/Model/Book.cs
namespace OneBeyondApi.Model
{
    public class Book
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public Author Author { get; set; }
        public BookFormat Format { get; set; }
        public string ISBN { get; set; }
    }
}

File: PHP/library-api-php/models/Book.php
<?php

class Book {
    public $id;
    public $title;
    public $authorId;
    public $format;
    public $isbn;

    public function __construct($id, $title, $authorId, $format, $isbn) {
        $this->id = $id;
        $this->title = $title;
        $this->authorId = $authorId;
        $this->format = $format;
        $this->isbn = $isbn;
    }
}
